### PR TITLE
Add 1.11 and k8s supported versions

### PR DESCRIPTION
**Pull request by ericvn**
Leaving the banner discussion in https://github.com/istio/istio.io/issues/10151 to another PR.

### CODE REVIEW

File: content/en/docs/releases/supported-releases/index.md [hunk 1]
```diff
@@ -54,13 +54,19 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
 | Version         | Currently Supported   | Release Date        | End of Life          | Supported Kubernetes Versions | Untested, but may work on Kubernetes Versions |
```
Comment (craigbox) [inline content/en/docs/releases/supported-releases/index.md:54]: "Untested, but may work" to fix the table width?

File: content/en/docs/releases/supported-releases/index.md [hunk 2]
```diff
@@ -54,13 +54,19 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
...
 |-----------------|-----------------------|---------------------|----------------------|-------------------------------|-----------------------------------------------|
 | master          | No, development only  |                     |                      |                               |                                               |
+| 1.11            | Yes                   | August 2021 (Expect)| ~Feb 2022 (Expected) | 1.19, 1.20, 1.21, 1.22        | 1.16, 1.17, 1.18                                    |
 | 1.10            | Yes                   | May 18, 2021        | ~Nov 2021 (Expected) | 1.18, 1.19, 1.20, 1.21        | 1.16, 1.17                                    |
 | 1.9             | Yes                   | February 9, 2021    | Aug 18, 2021         | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16                                    |
 | 1.8             | Yes                   | November 10, 2020   | May 12, 2021         | 1.16, 1.17, 1.18, 1.19        | 1.15                                          |
```
Comment (craigbox) [inline content/en/docs/releases/supported-releases/index.md:60]: Should 1.8 now say "no"?

Comment (ericvn) [reply content/en/docs/releases/supported-releases/index.md]: Good catch. Updated.

**Comment by ericvn** [comment]
Also, not sure if it makes sense to combine lines, like add 1.6 to the 1.5 line.

File: content/en/docs/releases/supported-releases/index.md [hunk 3]
```diff
@@ -51,15 +51,21 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
...
-| 1.6             | No                    | May 21, 2020        | November 23, 2020    | 1.15, 1.16, 1.17, 1.18        |                                               |
-| 1.5 and earlier | No                    |                     |                      |                               |                                               |
+| Version         | Currently Supported   | Release Date        | End of Life          | Supported Kubernetes Versions | Untested, but may work |
+|-----------------|-----------------------|---------------------|----------------------|-------------------------------|------------------------|
+| master          | No, development only  |                     |                      |                               |                        |
+| 1.11            | Yes                   | August 2021 (Expect)| ~Feb 2022 (Expected) | 1.19, 1.20, 1.21, 1.22        | 1.16, 1.17, 1.18       |
+| 1.10            | Yes                   | May 18, 2021        | ~Nov 2021 (Expected) | 1.18, 1.19, 1.20, 1.21        | 1.16, 1.17             |
```
Comment (craigbox) [inline content/en/docs/releases/supported-releases/index.md:58]: I assume we should add 1.22 to the "untested but may work" version, and maybe also add Kubernetes 1.21 to Istio 1.9 ?

File: content/en/docs/releases/supported-releases/index.md [hunk 4]
```diff
@@ -51,15 +51,21 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
...
+| 1.9             | Yes                   | February 9, 2021    | Aug 18, 2021         | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16             |
+| 1.8             | No                    | November 10, 2020   | May 12, 2021         | 1.16, 1.17, 1.18, 1.19        | 1.15                   |
+| 1.7             | No                    | August 21, 2020     | Feb 25, 2021         | 1.16, 1.17, 1.18              | 1.15                   |
+| 1.6             | No                    | May 21, 2020        | November 23, 2020    | 1.15, 1.16, 1.17, 1.18        |                        |
+| 1.5 and earlier | No                    |                     |                      |                               |                        |
+
+{{< warning >}}
+Kubernetes 1.22 removed some deprecated APIs and versions of Istio prior to 1.10.0 will no longer work. If you are
+upgrading your Kubernetes version, make sure that your Istio version is still supported.
```
[the remark this answers is not in the document]
Comment (craigbox) [reply content/en/docs/releases/supported-releases/index.md:67]: if we're going to cherrypick the banner we should also put this section (and the 1.22 is OK but untested on 1.10) in release-1.10.

File: content/en/docs/releases/supported-releases/index.md [hunk 5]
```diff
@@ -51,15 +51,21 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
...
+| master          | No, development only  |                     |                      |                               |                        |
+| 1.11            | Yes                   | August 2021 (Expect)| ~Feb 2022 (Expected) | 1.19, 1.20, 1.21, 1.22        | 1.16, 1.17, 1.18       |
+| 1.10            | Yes                   | May 18, 2021        | ~Nov 2021 (Expected) | 1.18, 1.19, 1.20, 1.21        | 1.16, 1.17             |
+| 1.9             | Yes                   | February 9, 2021    | Aug 18, 2021         | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16             |
+| 1.8             | No                    | November 10, 2020   | May 12, 2021         | 1.16, 1.17, 1.18, 1.19        | 1.15                   |
+| 1.7             | No                    | August 21, 2020     | Feb 25, 2021         | 1.16, 1.17, 1.18              | 1.15                   |
+| 1.6             | No                    | May 21, 2020        | November 23, 2020    | 1.15, 1.16, 1.17, 1.18        |                        |
+| 1.5 and earlier | No                    |                     |                      |                               |                        |
```
Comment (brian-avery) [inline content/en/docs/releases/supported-releases/index.md:63]: Why do we decide to draw the line here? Should it be 1.8 and earlier?

File: content/en/docs/releases/supported-releases/index.md (hunk 3 above)
Comment (ericvn) [reply content/en/docs/releases/supported-releases/index.md:58]: Looking at the jobs, 1.9 is only tested up to 1.19, and 1.10 (and 1.11) have not tested on 1.22.0 GA but still on 1.22.beta. But agree we should add it to 1.10 as well.

File: content/en/docs/releases/supported-releases/index.md [hunk 6]
```diff
@@ -51,15 +51,20 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
...
-| 1.10            | Yes                   | May 18, 2021        | ~Nov 2021 (Expected) | 1.18, 1.19, 1.20, 1.21        | 1.16, 1.17                                    |
-| 1.9             | Yes                   | February 9, 2021    | Aug 18, 2021         | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16                                    |
-| 1.8             | Yes                   | November 10, 2020   | May 12, 2021         | 1.16, 1.17, 1.18, 1.19        | 1.15                                          |
-| 1.7             | No                    | August 21, 2020     | Feb 25, 2021         | 1.16, 1.17, 1.18              | 1.15                                          |
-| 1.6             | No                    | May 21, 2020        | November 23, 2020    | 1.15, 1.16, 1.17, 1.18        |                                               |
-| 1.5 and earlier | No                    |                     |                      |                               |                                               |
+| Version         | Currently Supported   | Release Date        | End of Life          | Supported Kubernetes Versions | Untested, but may work |
```
Comment (ericvn) [inline content/en/docs/releases/supported-releases/index.md:54]: @craigbox I guess the "Untested, but may work" is probably incorrect. We actually test on those releases. Should we change to "Tested, but not supported"?

File: content/en/docs/releases/supported-releases/index.md (hunk 5 above)
Comment (ericvn) [reply content/en/docs/releases/supported-releases/index.md:63]: That's the same comment I made above with moving the cutoff up to 1.6 (as opposed to your 1.8). Will I think 1.8 would be the appropriate thing to do (keep the last line being a single `unsupported` line, I know we have many users on older versions and having the supported k8s versions in the table may make sense. Not sure if @therealmitchconnors has a suggestion on where the majority of users are.

Comment (craigbox) [reply content/en/docs/releases/supported-releases/index.md]: istr asking @jacob-delgado about this once?

File: content/en/docs/releases/supported-releases/index.md (hunk 6 above)
Comment (craigbox) [reply content/en/docs/releases/supported-releases/index.md]: If that's true, SGTM

File: content/en/docs/releases/supported-releases/index.md [hunk 7]
```diff
@@ -51,15 +51,19 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
...
-| 1.7             | No                    | August 21, 2020     | Feb 25, 2021         | 1.16, 1.17, 1.18              | 1.15                                          |
-| 1.6             | No                    | May 21, 2020        | November 23, 2020    | 1.15, 1.16, 1.17, 1.18        |                                               |
-| 1.5 and earlier | No                    |                     |                      |                               |                                               |
+| Version         | Currently Supported   | Release Date        | End of Life          | Supported Kubernetes Versions | Tested, but not supported |
+|-----------------|-----------------------|---------------------|----------------------|-------------------------------|---------------------------|
+| master          | No, development only  |                     |                      |                               |                           |
+| 1.11            | Yes                   | August 2021 (Expect)| ~Feb 2022 (Expected) | 1.19, 1.20, 1.21, 1.22        | 1.16, 1.17, 1.18          |
```
Comment (craigbox) [inline content/en/docs/releases/supported-releases/index.md:57]: (Expect) reads odd; either (Expected), or, I think it's fair to assume 1.11 will come out in August 2021.  There's a lot of days left even if the first date is missed :)

Comment (ericvn) [reply content/en/docs/releases/supported-releases/index.md]: Removed.

**Review by craigbox — APPROVED** [review]

**Comment by craigbox** [comment]
approved; it's LGTM as-is, but in case you still wanted to change it, put a WIP label on

**Comment by ericvn** [comment]
I'm fine letting this go in now. We can make any other changes when we fill in the exact 1.11 release date.

**Pull request merged**